src/sx/src/create.js

```diff
@@ -50,7 +51,14 @@ export default function create<T: SheetDefinitions>(
     `Function 'sx.create' cannot be called with empty stylesheet definition.`,
   );
 
-  const { hashRegistry } = styleCollector.collect(sheetDefinitions);
+  const { hashRegistry, styleBuffer } = styleCollector.collect(sheetDefinitions);
+
+  if (
+    isBrowser() &&
+    process.env.NODE_ENV !== 'test' // TODO: how to do it better (?)
```

Comment:
What's the issue with the test environment?

Reply:
It wants the element to be present somewhere in the document (see the invariant you commented on below).

Reply:
Right, so creating it if it does not exists would solve this as well.

Reply:
Most likely yes. I will look at it if I should do it now or in the next PR.